### PR TITLE
Update social icons in plugin list

### PR DESCRIPTION
**Pull request by tw2113**
This PR removes Google+ which no longer exists, and removes Twitter which is not functioning like it used to. Probably due to platform changes. I tried the same URL with x.com and it didn't fix anything.

### CODE REVIEW

File: includes/class-admin.php [hunk 1]
```diff
@@ -524,8 +524,6 @@ public function add_social_links( $links ) {
 
 		$social_share = esc_html__( 'Spread the word!', 'constant-contact-forms' );
 		$add_links[]  = '<a title="' . $social_share . '" href="https://www.facebook.com/sharer/sharer.php?u=' . rawurlencode( $site_link ) . '" target="_blank" rel="noopener noreferrer" class="dashicons-before dashicons-facebook"></a>';
-		$add_links[]  = '<a title="' . $social_share . '" href="https://twitter.com/home?status=' . $twitter_cta . ' ' . $site_link . '" target="_blank" rel="noopener noreferrer" class="dashicons-before dashicons-twitter"></a>';
```
Comment (ramsesdelr) [inline includes/class-admin.php:527]: I think you can replace this URL with: `https://twitter.com/intent/tweet?text=`, can you give it a try please?

**Review by ramsesdelr — CHANGES_REQUESTED** [review]
I left a comment regarding the twitter share link.

File: includes/class-admin.php (hunk 1 above)
Comment (tw2113) [reply includes/class-admin.php:527]: Almost, looks like it'll want to be `https://x.com/intent/post?text=` due to tragic loss of branding

Comment (tw2113) [reply includes/class-admin.php:527]: the CTA is already escaped via `esc_html__()`, but I can do so for the sitelink.

Comment (tw2113) [reply includes/class-admin.php:527]: This should all be resolved with https://github.com/WebDevStudios/constant-contact-forms/pull/678/commits/50c490273437a4f5bb3c731897fd24bec11213ad

**Review by ramsesdelr — APPROVED** [review]
Approved!

**Pull request merged**